econpizza/solvers/shooting.py

```diff
@@ -5,7 +5,9 @@
 import jax
 import jax.numpy as jnp
 from grgrjax import newton_jax_jit, val_and_jacfwd
+from ..config import enable_persistent_cache
 
+enable_persistent_cache()
```

Comment:
I don't understand why this is called here. Is there a reason why you don't just expose the function `enable_persistend_cache()` as a class method to config, and let the user call it whenever they feel like it? The workflow would then be:
```
from econpizza import config
config.enable_persistent_cache()
```
The variable `enable_persistent_cache` would then no longer be necessary.

Reply:
If this works, I think `enable_persistent_cache()` should have the optional argument to set the path to econpizza and jax cache folders.

Reply:
You are right, we should keep it similar to what jax does. So `ep.config.update("enable_persistent_cache", True)` as you put it in the description above seems to be a good choice. In the same way one could set custom cache folder paths.